dydxprotocol/v4-web
# chore: separating app UI configs slice

separate ui configs from configs slice, and run migration to redux persist

testing:
- locally change theme to light on main branch
- switch to this branch and see if it's still light
- restart chrome and check if it's still light
